tests/nl/homeassistant_HassGetState.yaml

```diff
@@ -1,2 +1,55 @@
 language: nl
-tests: []
+tests:
+  - sentences:
+      - "Wat is de buitentemperatuur"
+    intent:
+      name: HassGetState
+      slots:
+        name: "Buitentemperatuur"
+    response: "Buitentemperatuur is 15 °C"
+
+  - sentences:
+      - "Staat de slaapkamerlamp aan?"
+    intent:
+      name: HassGetState
+      slots:
+        name: "Slaapkamerlamp"
+        state: "on"
+    response: "Nee, slaapkamerlamp is off"
+
+  - sentences:
+      - "Staan er schakelaars aan in de keuken?"
+    intent:
+      name: HassGetState
+      slots:
+        area: "Keuken"
+        domain: "switch"
+        state: "on"
+    response: "Ja, Keukenschakelaar"
+
+  - sentences:
+      - "Zijn alle schakelaars aan?"
+    intent:
+      name: HassGetState
+      slots:
+        domain: "switch"
+        state: "on"
+    response: "Nee, Slaapkamerschakelaar"
```

Comment:
klinkt raar. Zou verwachten "Nee, niet de slaapkamerschakelaar"

Reply:
Lidwoorden toevoegen is wat lastig, want bijvoorbeeld `slaapkamerschakelaartje` zou `het` moeten zijn, maar ik zal `niet` toevoegen.